pkg/apis/istio/v1alpha1/fixup_structs/main.go

```diff
@@ -47,6 +47,15 @@ const (
 	goTypeToken = "// GOTYPE: "
 )
 
+var (
+	nameMapping = map[string]string{
```

Comment:
Could you put a comment with the reason for this mapping?

Reply:
added